### DISCUSSION TITLE
Does emissive can  illuminate other objects？

### DISCUSSION THREAD

**Discussion by dugufly** [google/filament#6092] [Q&A] [answered]
I added two objects to the scene, one of them is emissive, no other light sources are added to the scene, but the other object is not illuminated. Can emissive illuminate other objects like other light sources?

**Comment by romainguy** [comment, marked as answer]
Emissive doesn't illuminate other objects, you have to put a light next to your emissive surface. It would work if we had dynamic GI but that's expensive for our targets (mobile).

**Reply by dugufly** [reply]
I want to achieve a lighting effect similar to a Bar light. The area light  is not supported in filament, and emissive also cannot be achieved . 
Is there any other way to approximate this effect?

**Reply by romainguy** [reply]
Just like they do in many games: put a point or spot light on your neon (or several).

**Reply by dugufly** [reply]
Does the engine plan to support area light in the future? It seems that LTC algorithm can achieve real-time.

**Reply by romainguy** [reply]
Maybe. LTC can achieve real-time but it remains very expensive for mobile. There are cheaper alternatives though, although not quite as good.

**Reply by dugufly** [reply]
Thank you!
